pkg/runTest.py

```diff
@@ -93,8 +94,28 @@ def run(**kwargs: dict[str, Any]) -> dict[str, Any]: #pylint: disable = R0914
             )
         testname, result_data, test_flag = algorithm.output(kwargs["output_format"])
         result_output[testname] = result_data
+        # Query with JSON
+        regression_data = []
+        if test_flag:
+            testname, result_data, test_flag = algorithm.output(cnsts.JSON)
+            prev_ver = None
+            bad_ver = None
+            for result in json.loads(result_data):
+                if result["is_changepoint"]:
+                    bad_ver = result["ocpVersion"]
+                else:
+                    prev_ver = result["ocpVersion"]
+                if prev_ver is not None and bad_ver is not None:
+                    regression_data.append({
+                        "prev_ver": prev_ver,
+                        "bad_ver": bad_ver,
+                        "prs": Utils().sippy_pr_diff(prev_ver, bad_ver)
+                    })
+                    prev_ver = None
```

Comment:
we can  move these vars to set to None starting at #104 and get rid of the ones at #101, I beleive.